test/blas_test.hpp

```diff
@@ -125,19 +125,33 @@ static inline scalar_t random_scalar(scalar_t rangeMin, scalar_t rangeMax) {
 }
 
 /**
- * @fn random_data
  * @brief Generates a random vector of scalar values, using a uniform
  * distribution.
+ * @param vec Input vector to fill
+ * @param rangeMin Minimum value for the uniform distribution
+ * @param rangeMax Maximum value for the uniform distribution
  */
 template <typename scalar_t>
-static inline void fill_random(std::vector<scalar_t> &vec) {
+static inline void fill_random_with_range(std::vector<scalar_t> &vec,
+                                          scalar_t rangeMin,
+                                          scalar_t rangeMax) {
   for (scalar_t &e : vec) {
-    e = random_scalar(scalar_t{-2}, scalar_t{5});
+    e = random_scalar(rangeMin, rangeMax);
   }
 }
 
 /**
- * @breif Fills a lower or upper triangular matrix suitable for TRSM testing
+ * @fn random_data
```

Comment:
This should probably be removed as the name doesn't match, and I believe `fn` should only be used when the comment block does not directly proceed the function it is supposed to document ([see here](https://www.doxygen.nl/manual/commands.html#cmdfn))

Reply:
Thanks @Bensuo , I removed it.

Reply:
Thanks @Bensuo , removed it.